zubhub_frontend/zubhub/src/views/create_project/CreateProject.jsx

```diff
@@ -642,7 +642,12 @@ function CreateProject(props) {
                             color="textSecondary"
                             className={classes.customLabelStyle}
                           >
-                            <Box className={classes.fieldNumberStyle}>6</Box>
+                            <Box
+                              className={classes.fieldNumberStyle}
+                              id="sixCenter"
```

Comment:
instead of id, use className like below and define the class in useStyles

```suggestion
                              className={classnames([classes.fieldNumberStyle, classes.circleCenteredText])}
```